packages/plugins/src/plugins/diagnostic-fix.plugin.ts

```diff
@@ -59,20 +59,30 @@ export class DiagnosticFixPlugin extends Plugin {
       // TODO: User should be able to choose one of the fixes form this list in interactive mode
       const fix = fixes.shift()!;
 
-      for (const fileTextChange of fix.changes) {
-        let text = this.service.getFileText(fileTextChange.fileName);
+      if (isInstallPackageCommand(fix)) {
+        try {
+          await this.service.getLanguageService().applyCodeActionCommand(fix.commands[0]);
+        } catch (e) {
+          // swallow the error we couldn't install the dep or it didn't actually exist
+        }
+      } else {
```

Comment:
I think we need to run `fix.changes` after commands are applied because after the package is installed it can modify files with `import` statements.